src/Config.php

```diff
@@ -192,15 +212,18 @@ public function isAlias(string $name): bool
      * {@inheritDoc}
      *
      * @see ConfigInterface::getConfiguredTypeNames()
+     *
+     * @return list<string>
      */
     public function getConfiguredTypeNames(): array
     {
-        return array_keys($this->types);
+        return array_map('strval', array_keys($this->types));
```

Comment:
Make sure this is actually a string list. PHP always converts integerish strings to an `int` when they are used as array key, so this is actually required: https://3v4l.org/O2Wd9